web/src/components/filters/filter-action-links.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { Button } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+
+export interface FilterActionLinksProps {
+  isEmpty: boolean;
+  isDefault: boolean;
+  resetFilters: () => void;
+  clearFilters: () => void;
+}
+
+export const FilterActionLinks: React.FC<FilterActionLinksProps> = ({
+  isEmpty,
+  isDefault,
+  resetFilters,
+  clearFilters
+}) => {
+  const { t } = useTranslation('plugin__netobserv-plugin');
+
+  return (
+    <>
+      {!isDefault && (
```

Comment:
It feels strange to see this reset button when you don't have any filter selected

![image](https://user-images.githubusercontent.com/91894519/197991209-985a0046-ab30-4ba9-96d1-d0b1757a6290.png)

What about hiding it when you don't have any ?
```suggestion
      {!isEmpty && !isDefault && (
```

Reply:
Or maybe move the reset button inside the "Filters" dropdown ?

Reply:
Also, when frontend config is not provided, I still see the reset button
![image](https://user-images.githubusercontent.com/91894519/197996164-3dc364df-23b4-445e-95b7-648e5ac8da4f.png)

Reply:
> It feels strange to see this reset button when you don't have any filter selected

@andrew-ronaldson any thought on that ?